env.example

```diff
@@ -4,21 +4,22 @@ SIGNER_ACCOUNT_ADDRESS=<signer-account-address>
 SIGNER_ACCOUNT_PRIVATE_KEY=<signer-account-private-key>
 SLOT_ID=<slot-id>
 SNAPSHOT_CONFIG_REPO=https://github.com/powerloom/snapshotter-configs
-SNAPSHOT_CONFIG_REPO_BRANCH=eth_uniswapv3_lite
+SNAPSHOT_CONFIG_REPO_BRANCH=<snapshot-config-repo-branch>
 SNAPSHOTTER_COMPUTE_REPO=https://github.com/powerloom/snapshotter-computes
-SNAPSHOTTER_COMPUTE_REPO_BRANCH=eth_uniswapv3_lite
+SNAPSHOTTER_COMPUTE_REPO_BRANCH=<snapshotter-compute-repo-branch>
 DOCKER_NETWORK_NAME=
 PROST_RPC_URL=https://rpc-prost1m.powerloom.io
-PROTOCOL_STATE_CONTRACT=0xE88E5f64AEB483d7057645326AdDFA24A3B312DF
-DATA_MARKET_CONTRACT=0x0C2E22fe7526fAeF28E7A58c84f8723dEFcE200c
-NAMESPACE=UNISWAPV2
+PROTOCOL_STATE_CONTRACT=0xF68342970beF978697e1104223b2E1B6a1D7764d
+DATA_MARKET_CONTRACT=<data-market-contract>
```

Comment:
If we're overriding all these values in `build.sh` based on user selection, do we need them here?

Reply:
Yeah I wouldn't say it is necessary to put even the protocol state contract in here. Will review and push changes.

Reply:
11f1a7ef addresses this